test/refactor_nrepl/ns/suggest_aliases_test.clj

```diff
@@ -21,6 +21,22 @@
     'unit.foo   true
     'foo.unit   true))
 
+(deftest readable-as-symbol?
+  (are [input expected] (is (= expected
+                               (sut/readable-as-symbol? input)))
+    "test"     true
+    "a.test"   true
+    "a-test"   true
+    "b.a-test" true
+    "ok then"  true
```

Comment:
while I don't expect `ok then` be produced irl, it is a good edge case to think of.

If it's at hand, the result should be `false` because a symbol with a space can be surprising / unusable client-side.

Then again I don't expect it to be a real problem, so feel free not to spend time in it

(in which case, remove this line so that we remain agnostic)

Reply:
Good point, took it out.